tooling/build-types-and-manifests.ts

```diff
@@ -53,20 +53,13 @@ addManifest(
   'thunderbird-consumer',
   manifest => {
     manifest.browser_specific_settings.strict_min_version = '102.0';
-    manifest.browser_action.default_title = 'FlowCrypt Encryption for Thunderbird';
+    manifest.browser_action.default_title = 'FlowCrypt';
     manifest.name = 'FlowCrypt Encryption for Thunderbird';
-    manifest.description = 'Secure end-to-end encryption with FlowCrypt'; // needs to updated later
+    manifest.description = 'Simple end-to-end encryption to secure email and attachments on Thunderbird';
+    manifest.permissions = [...manifest.permissions, 'compose'];
     manifest.compose_action = {
       default_title: 'FlowCrypt', // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
let's call it `Secure compose`, so it'll be the same as in the browser extension